File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/weather/model/Units.java

package restaurant.sa.com.sarestaurant.appview.weather.model;

import com.google.gson.annotations.SerializedName;
public class Units {

    @SerializedName("distance")
    private String mDistance;
    @SerializedName("pressure")
    private String mPressure;
    @SerializedName("speed")
    private String mSpeed;
    @SerializedName("temperature")
    private String mTemperature;

    public String getDistance() {
        return mDistance;
    }

    public void setDistance(String distance) {
        mDistance = distance;
    }

    public String getPressure() {
        return mPressure;
    }

    public void setPressure(String pressure) {
        mPressure = pressure;
    }

    public String getSpeed() {
        return mSpeed;
    }

    public void setSpeed(String speed) {
        mSpeed = speed;
    }

    public String getTemperature() {
        return mTemperature;
    }

    public void setTemperature(String temperature) {
        mTemperature = temperature;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/weather/model/Units.kt
package restaurant.sa.com.sarestaurant.appview.weather.model

import com.google.gson.annotations.SerializedName

class Units {

    @SerializedName("distance")
    var distance: String? = null
    @SerializedName("pressure")
    var pressure: String? = null
    @SerializedName("speed")
    var speed: String? = null
    @SerializedName("temperature")
    var temperature: String? = null

}
